src/app/body/filter-page/edit-filter/edit-filter.component.css

```diff
@@ -11,16 +11,109 @@
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
 * See the MIT License for more details. 
 ***********************************************************/
+#openEditPropertiesContainer,
+.modal-content{
+  background-color: var(--secondary-bg);
+  color: var(--button-text);
+}
+
+.btn {
+    color: var(--text);
+}
+
+
 #button {
     background-color: var(--primary-color);
     color: white;
 }
 
+.btn {
+    color: var(--text);
+}
```

Comment:
you are using .btn{} twice in line 20 and 30

Reply:
okay I will make the change